dts/bindings/sensor/ti,ina237.yaml

```diff
@@ -15,13 +15,80 @@ compatible: "ti,ina237"
 include: ti,ina23x-common.yaml
 
 properties:
+  config:
+    type: int
+    deprecated: true
+    default: 0x0000
+    description: |
+      Value of the configuration register
+      e.g shunt voltage and bus voltage ADC conversion
+      times and averaging, operating mode for INA230 or
+      delay for initial ADC conversion, shunt full scale range
+      for INA237.
+
+      Deprecated, please use new properties instead.
+
   adc-config:
     type: int
-    required: true
+    deprecated: true
+    default: 0x0000
     description: |
       Value of the ADC configuration register (ADC conversion times,
       averaging, operating mode and etc).
 
+      Deprecated, please use new properties instead.
+
   alert-config:
     type: int
     description: Diag alert register, default matches the power-on reset value
+
+  adc-mode:
+    type: string
+    description: |
+      ADC channel conversion configuration.
+    default: "Temperature, bus, and shunt voltage continuous"
+    enum:
+      - "Shutdown single shot"
+      - "Bus Voltage single shot"
+      - "Shunt Voltage single shot"
+      - "Bus and Shunt Voltage single shot"
+      - "Temperature Single shot"
+      - "Temperature and bus voltage single shot"
+      - "Temperature and shunt voltage single shot"
+      - "Temperature, bus, and shunt voltage single shot"
+      - "Shutdown continuous"
+      - "Bus voltage continuous"
+      - "Shunt voltage continuous"
+      - "Bus and shunt voltage continuous"
+      - "Temperature continuous"
+      - "Temperature and bus voltage continuous"
+      - "Temperature and shunt voltage continuous"
+      - "Temperature, bus, and shunt voltage continuous"
+
+  vbus-conversion-time-us:
+    type: int
+    description: |
+      Vbus conversion time in microseconds.
+    default: 1052
```

Comment:
Property defaults need a justification in the description. Is this register reset value?

Reply:
Changes will be added in the next push to the PR.